homeassistant/components/russound_rio/media_player.py

```diff
@@ -120,9 +123,20 @@ class RussoundZoneDevice(MediaPlayerEntity):
 
     def __init__(self, zone: Zone, sources: dict[int, Source]) -> None:
         """Initialize the zone device."""
-        super().__init__()
+        self._controller = zone.controller
         self._zone = zone
         self._sources = sources
+        self._attr_unique_id = str(self._zone)
+        self._attr_device_info = DeviceInfo(
+            identifiers={
+                # Use MAC address of Russound device as identifier
```

Comment:
You can also add it to connections so the entities will merge with discovered devices from your router etc